docs/pages/concepts/offloading/namespace-replication.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Namespace replication
+weight: 2
+---
+
+### The Liqo Namespace Model
+
+The Liqo resource replication model is composed of several concurring logics responsible for different kind of objects to replicate.
+One important example is represented by namespaces. 
+In particular, namespace replication across multiple clusters implements a key mechanism to extend seamleassly a cluster to some others.
+
+In the Liqo model, the namespace replication corresponds to a standard v1.Namespace and an associated [NamespaceOffloading](#) CR.
+The NamespaceOffloading resource allows to select the clusters which are the clusters where the application contained inside the namespace can be scheduled.
+
+In [a dedicated usage section](/usage/namespace_offloading), you can learn how to define and use a NamespaceOffloading resource activating the replication of the local namespace on peered clusters.
+
+In this section, you may discover how Liqo namespace model works under the hood, undestanding which resources and controllers are activated when a new namespaceOffloading is manipulated.
+
+#### Resources
+
+Liqo namespace replication involves instances of two kinds of CRDs:
+
+* First, the initial trigger is represented by the manipulation of a **NamespaceOffloading** (or the labelling of a namespace as explained in [the usage section](/usage/namespace_offloading)).
+  On the one hand, the *NamespaceOffloading* spec describes the properties of the replication, such as the name of the replicated namespaces.
+  On the other hand, the status collects the information about the actual status of remote namespaces (i.e. if the replication has succeeded).
+* Second, **NamespaceMap** resources contain the list of namespaces associated to a specific node.
```

Comment:
```suggestion
* **NamespaceMap**: this CR contains the list of namespaces associated to a specific node.
```

Reply:
I don't know what "list of namespaces associated to a specific node" means.

Reply:
I have tried to clarify the concept, thank you.